designer/client/field-edit.tsx

```diff
@@ -46,10 +46,10 @@ export function FieldEdit() {
           rows={2}
           label={{
             className: "govuk-label--s",
-            children: ["Help Text (optional)"],
+            children: ["Help text (optional)"],
           }}
           hint={{
-            children: ["Text can include HTML"],
+            children: [i18n("fieldeditors.helptext")],
```

Comment:
do you mind camel casing so it's easier to read these please "fieldEditor.helpText"